setup.py

```diff
@@ -31,6 +31,7 @@
     "pennylane>=0.32",
     f"jax=={jax_version}",
     f"jaxlib=={jax_version}",
+    "pennylane_autograph==2.15.0",
```

Comment:
@dime10 any particular reason for pinning here? the general rule of thumb is to be as unrestrictive as possible in `setup.py`, to avoid issues where dependencies cannot be resolved on a users local environment when `pip` installing

Reply:
Well we know it works with that version, other versions might not work. Well I suppose since they don't exist yet we might as well remove the pinning.

Reply:
Fixed, it now just says `"diastatic-malt"`.